native/src/main/java/io/ballerina/persist/cmd/PersistCmd.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.persist.cmd;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.persist.PersistToolsConstants;
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+
+
+/**
+ * Class to implement "persist" command for ballerina.
+ */
+@CommandLine.Command(
+        name = "persist",
+        description = "generate database configurations.",
+        subcommands = {Init.class}
+        )
+
+public class PersistCmd implements BLauncherCmd {
+
+    private static final PrintStream outStream = System.out;
+
+    private CommandLine parentCmdParser;
+
+    @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
+    private boolean helpFlag;
+
+    @Override
+    public void execute() {
+        if (helpFlag) {
+            outStream.println("Hello persist");
```

Comment:
we need to give a proper help message to the user. https://github.com/ballerina-platform/module-ballerina-grpc/blob/master/native/src/main/java/io/ballerina/stdlib/grpc/protobuf/cmd/GrpcCmd.java#L131

Reply:
check this later

Reply:
updated in  [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)